Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -26,90 +26,130 @@ Meaning of columns:
+| `IfcCartesianTransformationOperator2DnonUniform` | :heavy_check_mark:   | `PlacementConverter`       | :x:                   | -     |
+| `IfcCircle`                           | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcCircularArcSegment2D`             | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcClosedShell`                      | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | -     |
 | `IfcClothoid`                         | :x:                  | `CurveConverter`           | -                     | [#304](https://github.com/tumcms/Open-Infra-Platform/issues/304) |
-| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
 | `IfcCompositeCurveOnSurface`          | :x:                  | -                          | -                     | -     |
-| `IfcCompositeCurveSegment`            | :x:                  | `CurveConverter`           | -                     | -     |
-| `IfcCsgSolid`                         | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
-| `IfcCurveSegment`                     | :x:                  | `CurveConverter`           | -                     | -     |
-| `IfcDirection`                        | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
-| `IfcEllipse`                          | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
-| `IfcExtrudedAreaSolid`                | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcCompositeCurveSegment`            | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcConnectedFaceSet`                 | :x:                  | 'RepresentationConverter'  | -                     | -     |
+| `IfcCsgSolid`                         | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | [#356](https://github.com/tumcms/Open-Infra-Platform/issues/356) |
+| `IfcCurveBoundedPlane`                | :heavy_check_mark:   | `FaceConverter`            | :x:                   | -     |
+| `IfcCurveBoundedSurface`              | :heavy_check_mark:   | `FaceConverter`            | :x:                   | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150) |
+| `IfcCurveSegment`                     | :x:                  | `CurveConverter`           | -                     | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150) |
+| `IfcCylindricalSurface`               | :x:                  | `CurveConverter`           | -                     | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150) |
```

Comment:
- I'm not sure that CylindricalSurface would be covered by CurveConverter.
- are these really both included in #150 ?

Reply:
Resolved 6662fbd